src/main/java/frc/robot/commands/TeleopCommand.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.io.Controls;
+import frc.robot.subsystems.drive.Drive;
+
+public class TeleopCommand extends CommandBase {
+
+    private Drive drive;
+    private Controls controls;
+    private double DEADZONE = 0.11;
+    private boolean resetEncoders;
+  
+    public TeleopCommand(Controls controls, Drive drive) {
+        this.controls = controls;
+        this.drive = drive;
+    }
+
+    // Called just before this Command runs the first time
+    @Override
+    public void initialize() {
+        System.out.println("Initialized");
+
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    @Override
+    public void execute() {
+        if (controls.getRightX(DEADZONE) == 0.0) {
```

Comment:
Very rarely is comparing a floating point value with == acceptable.  For one, unless you are explicitly setting the value, floating points almost never equal whole numbers.  This is one of the few cases where that is so.  Second, == comparison for a floating point is computationally expensive and should be avoided if possible.  This can be left as is, but please keep that in mind when using floating point comparisons in the future.